### ISSUE TITLE
[BUG] `iohub info` does not work with reconstructed hcs plates

### ISSUE THREAD

**Issue by talonchandler** [mehta-lab/recOrder#407]
After the recent #402 merge, @edyoshikun noticed that `iohub info` does not work at the plate level. 

e.g.
```
iohub info test-output9.zarr/
Reading file:	 /Users/talon.chandler/Desktop/test-data/recOrder/test-output9.zarr
Error: No compatible dataset is found.
```

**Comment by edyoshikun** [comment]
I also realized that if you drag and drop the plate level to `napari-ome-zarr` it fails to open. However, if you drag it at a position level, it opens fine. This is probably some issue with metadata as @talonchandler pointed out.

**Comment by talonchandler** [comment]
I just looked closer and found that I'm not yet using @edyoshikun's `create_empty_zarr`, so it makes sense that I don't have a valid plate-level zarr.

So the `main` branch's CLI is standardized, but not yet compatible with plate-level zarrs. I think we're moving step-by-step in the right direction here, though, because the GUI is still working and it's only compatible with fov-level zarrs. 

Upcoming work:
- [ ] Fix #396 to prepare for `apply_inv_tf` multiprocessing
- [ ] Fix this issue by supporting multiprocessing...likely modifying #392

**Comment by talonchandler** [comment]
After further discussion with @edyoshikun, I think I can fix this issue by itself using `create_empty_zarr`, and we can leave #392 and #396 until after 0.4.0.
